magento/magento2
# #17488 Fix Authenticating a customer via REST API does not update the last logged in data

### Description

Fixed Last Logged In date when we authenticate a customer via REST API.
### Fixed Issues (if relevant)

magento/magento2#17488: Authenticating a customer via REST API does not update the last logged in data #17488
### Manual testing scenarios


1. Create a new customer account
2. Login with that customer using the default Magento frontend
3. View the customer in the Magento admin area, and note that the Last Logged In value has been updated
4. Request a customer token via this API endpoint: /rest/V1/integration/customer/token

http://{{base_url}}/rest/V1/integration/customer/token/?username=examplel@krishtechnolabs.com&password=examplel@123

### Contribution checklist
